pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -1,12 +1,24 @@
-{ callPackage }: builtins.mapAttrs (pname: attrs: callPackage ./generic.nix (attrs // { inherit pname; })) {
-  signal-desktop = {
+{ callPackage, stdenv }: builtins.mapAttrs (pname: attrs: callPackage ./generic.nix (attrs // { inherit pname; })) {
+  signal-desktop =
+    let
+      version = "6.40.0";
```

Comment:
Signal releases very frequently, we might have to make this two links in the future. It would be also nice to have an automated update script for the aarch64 deb archive as well as this breaks now my normal automation (nix-update).

Reply:
Also please add yourself as a maintainer as I cannot test the arm64 port.